showcase/app/components/shw/flex/index.gts

```diff
@@ -5,6 +5,9 @@
 
 import Component from '@glimmer/component';
 import { htmlSafe } from '@ember/template';
+import { hash } from '@ember/helper';
+import Item from './item';
+import Label from '../label';
```

Comment:
[chore] better to preserve the namespacing, so when we look at the code we know where a component instance comes from (`<Label>` vs `<ShwLabel>`, `<Item>` vs `<ShwFlexItem>`)
```suggestion
import ShwFlexItem from './item';
import ShwLabel from '../label';
```